netbox_acls/filtersets.py

```diff
@@ -151,7 +167,12 @@ def search(self, queryset, name, value):
         """
         Override the default search behavior for the django model.
         """
-        return queryset.filter(description__icontains=value)
+        query = (
+            Q(access_list__name__icontains=value)
+            | Q(index__icontains=value)
+            | Q(action__icontains=value)
+        )
```

Comment:
Shouldn't we still keep search by description here?
```python
        query = (
            Q(access_list__name__icontains=value)
            | Q(description__icontains=value)  # here
            | Q(index__icontains=value)
            | Q(action__icontains=value)
        )
```

Cause, like you've mentioned, that error happens since [`AccessList`](https://github.com/netbox-community/netbox-acls/blob/685a91bb07c0e54d4a59abee87d8827964ca60ac/netbox_acls/models/access_lists.py#L29) doesn't have `description` attribute. But [`ACLRule` does](https://github.com/netbox-community/netbox-acls/blob/685a91bb07c0e54d4a59abee87d8827964ca60ac/netbox_acls/models/access_list_rules.py#L38).